Formula/bedops.rb

```diff
@@ -0,0 +1,28 @@
+class Bedops < Formula
+  desc "Set and statistical operations on genomic data of arbitrary scale"
+  homepage "https://github.com/bedops/bedops"
+  url "https://github.com/bedops/bedops/archive/v2.4.30.tar.gz"
+  sha256 "218e0e367aa79747b2f90341d640776eea17befc0fdc35b0cec3c6184098d462"
+
+  head "https://github.com/bedops/bedops.git"
+
+  needs :cxx11
+
+  def install
+    ENV.O3
```

Comment:
why? The default is -Os which should be used unless there's a bug

Reply:
Long ago in Homebrew/homebrew-science#726 it was deemed necessary, but no explanation was given. Seems to work fine without it.